app/services/process_feed_service.rb

```diff
@@ -178,6 +172,16 @@ def mentions_from_xml(parent, xml)
       end
     end
 
+    def account_from_href(href)
+      url = Addressable::URI.parse(href)
+
+      if TagManager.instance.web_domain?(url.host)
+        Account.find_local(url.path.gsub('/users/', ''))
```

Comment:
Is this one of those cases where Rubocop would tell us to use .delete instead of .gsub?